### ISSUE TITLE
Docs advocate monkeypatching Basic

### ISSUE THREAD

**Issue by asmeurer** [sympy/sympy#5645]
```
At http://docs.sympy.org/0.7.0/modules/printing.html?highlight=printer#module-sympy.printing.printer , it says:

# Override the __str__ method of to use CustromStrPrinter
Basic.__str__ = lambda self: CustomStrPrinter().doprint(self)

I think we shouldn't be advocating to monkeypatch Basic in our docs. This should be rewritten to do it another way.
```

Original issue for #5645: http://code.google.com/p/sympy/issues/detail?id=2546
Original author: https://code.google.com/u/asmeurer@gmail.com/
Original owner: https://code.google.com/u/asmeurer@gmail.com/

**Comment by eric-wieser** [comment]
This was fixed in #12741

**Comment by oscarbenjamin** [comment]
Thanks for pointing that out
